### DISCUSSION TITLE
Add a colored background to the card

### DISCUSSION THREAD

**Discussion by suxlala** [punxaphil/custom-sonos-card#559] [Ideas]
Adding a blurred image of the current song's album art to the background would make the card more interesting. The card would change the background color constantly, in alignment with the color palette of the album art displayed, similar to some streaming audio apps, i.e. Apple Music.

**Comment by punxaphil** [comment]
Discussion moved here: https://github.com/punxaphil/custom-sonos-card/issues/564
